### PR TITLE
Fix the Jenkins Solution Template to allow referencing existing resources

### PR DESCRIPTION
**Pull request by clguiman**
I forgot to pass the resource group, so the user can reference an existing storage account or public IP.

### CODE REVIEW

File: solution_template/nested/publicIP-new.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "outputs": {
     "fqdn": {
       "type": "string",
-      "value": "[reference(resourceId('Microsoft.Network/publicIPAddresses',parameters('publicIPName')),'2016-03-30').dnsSettings.fqdn]"
+      "value": "[reference(resourceId(parameters('publicIPResourceGroup'),'Microsoft.Network/publicIPAddresses',parameters('publicIPName')),'2016-03-30').dnsSettings.fqdn]"
```
Comment (ejizba) [inline solution_template/nested/publicIP-new.json]: Should the resourceGroup be used even for the new public IP? Same for the storage account

Comment (clguiman) [reply solution_template/nested/publicIP-new.json]: For new resources the resource group will be the same as for all the other resources. I left it there so  the output line can be the same in both templates.
I can remove it

Comment (ejizba) [reply solution_template/nested/publicIP-new.json]: As long as it doesn't break :)

**Review by ejizba — APPROVED** [review]

File: solution_template/nested/publicIP-new.json (hunk 1 above)
Comment (clguiman) [reply solution_template/nested/publicIP-new.json]: You're right, better to be safe :) 
I just pushed a new commit that doesn't reference the resource group for the '-new' templates

**Pull request merged**